docs/faq.yaml

```
@@ -1,3 +1,14 @@
+- tags:
+  - kratos
+  - secrets
+  - config
+  q: |
+    Is it possible to pass multiple system.secret values through the ENV, or do you
+    have to use a config file for that?
+  a: |
+    Yes, you can set them as environmental variable, you just have to separate them
```

```suggestion
    You can set them as environmental variable, you just have to separate them
```
Answering an or-question with "yes"? :stuck_out_tongue_winking_eye: